app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name">My Application</string>
-    <string name="web_client_id">957415054330-j5fa1cl19qlthoshv7l2cccqv5v4icj4.apps.googleusercontent.com</string>
+    <string name="web_client_id">1043895872028–8m8f4kpcmqe60ofkef734vbgqph8g5ut.apps.googleusercontent.com</string>
```

Comment:
These changes are already merged by Nik, maybe do one more git pull to sync your changes with his.

Reply:
Actually, that line is already in sync with what Nikhen made. It is in this PR : https://github.com/swent-sp-2024-event-radar/event-radar-app/pull/61/files#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103 which still needs to merge into main. Maybe I should have taken the lines in main instead of Nikhen's (I did this because CI testrunner had a failure telling me there was a duplicate of "client id")